src/sentry/static/sentry/app/components/group/sidebar.jsx

```diff
@@ -78,6 +78,24 @@ const GroupSidebar = createReactClass({
         });
       },
     });
+
+    // Fetch the top values for the current group's top tags.
+    this.api.request(`/issues/${group.id}/tags/`, {
+      query: _.pickBy({
```

Comment:
Is the `pickBy` here necessary?

Reply:
just a way to remove the `environment` param from the list if it's not defined. Maybe too clever?